### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jsturdy**
## Description

* Replace all `localArgs` initialization with `GETLOCALARGS`
* Clean up some (not all) formatting mismatches
* Close all RPC callbacks with `rtxn.abort()`

### Types of changes
- [x] Cosmetic/cleanup

## Motivation and Context
Needed to happen, did it now to just be done with it

### CODE REVIEW

File: src/calibration_routines.cpp [hunk 1]
```diff
@@ -1193,15 +1155,15 @@ std::vector<uint32_t> dacScanLocal(localArgs *la, uint32_t ohN, uint32_t dacSele
     //Set the VFATs into Run Mode
     broadcastWriteLocal(la, ohN, "CFG_RUN", 0x1, mask);
     LOGGER->log_message(LogManager::INFO, stdsprintf("VFATs not in 0x%x were set to run mode", mask));
-    std::this_thread::sleep_for(std::chrono::seconds(1)); //I noticed that DAC values behave weirdly immediately after VFAT is placed in run mode (probably voltage/current takes a moment to stabalize)
+    std::this_thread::sleep_for (std::chrono::seconds(1)); //I noticed that DAC values behave weirdly immediately after VFAT is placed in run mode (probably voltage/current takes a moment to stabalize)
```
Comment (mexanick) [inline src/calibration_routines.cpp]: automated search/replace? ;)

File: src/calibration_routines.cpp [hunk 2]
```diff
@@ -1138,7 +1100,7 @@ std::vector<uint32_t> dacScanLocal(localArgs *la, uint32_t ohN, uint32_t dacSele
     //Check which VFATs are sync'd
     uint32_t notmask = ~mask & 0xFFFFFF; //Inverse of the vfatmask
     uint32_t goodVFATs = vfatSyncCheckLocal(la, ohN);
-    if( (notmask & goodVFATs) != notmask){
+    if ( (notmask & goodVFATs) != notmask) {
```
Comment (mexanick) [inline src/calibration_routines.cpp:1103]: extra space between opening parentheses

File: src/calibration_routines.cpp [hunk 3]
```diff
@@ -915,35 +882,35 @@ void checkSbitMappingWithCalPulse(const RPCMsg *request, RPCMsg *response){
...
+    if ( fw_version_check("checkSbitRateWithCalPulse", la) < 3) {
         LOGGER->log_message(LogManager::ERROR, "checkSbitRateWithCalPulse is only supported in V3 electronics");
         sprintf(regBuf,"checkSbitRateWithCalPulse is only supported in V3 electronics");
         la->response->set_string("error",regBuf);
         return;
     }
 
     uint32_t goodVFATs = vfatSyncCheckLocal(la, ohN);
-    if( (notmask & goodVFATs) != notmask){
+    if ( (notmask & goodVFATs) != notmask) {
```
Comment (mexanick) [inline src/calibration_routines.cpp:907]: extra space between parentheses

File: src/calibration_routines.cpp [hunk 4]
```diff
@@ -915,35 +882,35 @@ void checkSbitMappingWithCalPulse(const RPCMsg *request, RPCMsg *response){
...
 
-void checkSbitRateWithCalPulseLocal(localArgs *la, uint32_t *outDataCTP7Rate, uint32_t *outDataFPGAClusterCntRate, uint32_t *outDataVFATSBits, uint32_t ohN, uint32_t vfatN, uint32_t mask, bool useCalPulse, bool currentPulse, uint32_t calScaleFactor, uint32_t waitTime, uint32_t pulseRate, uint32_t pulseDelay){
+void checkSbitRateWithCalPulseLocal(localArgs *la, uint32_t *outDataCTP7Rate, uint32_t *outDataFPGAClusterCntRate, uint32_t *outDataVFATSBits, uint32_t ohN, uint32_t vfatN, uint32_t mask, bool useCalPulse, bool currentPulse, uint32_t calScaleFactor, uint32_t waitTime, uint32_t pulseRate, uint32_t pulseDelay)
+{
     //Determine the inverse of the vfatmask
     uint32_t notmask = ~mask & 0xFFFFFF;
 
     char regBuf[200];
-    if( fw_version_check("checkSbitRateWithCalPulse", la) < 3){
+    if ( fw_version_check("checkSbitRateWithCalPulse", la) < 3) {
```
Comment (mexanick) [inline src/calibration_routines.cpp:899]: extra space after opening bracket

File: src/daq_monitor.cpp [hunk 5]
```diff
@@ -312,22 +277,21 @@ void getmonGBTLink(const RPCMsg *request, RPCMsg *response)
   }
 
   bool doReset = false;
-  if(request->get_key_exists("doReset") ) {
+  if (request->get_key_exists("doReset") ) {
```
Comment (mexanick) [inline src/daq_monitor.cpp:280]: extra spaces between closing brackets

File: src/daq_monitor.cpp [hunk 6]
```diff
@@ -336,23 +300,22 @@ void getmonOHLink(const RPCMsg *request, RPCMsg *response)
...
```
Comment (mexanick) [inline src/daq_monitor.cpp:303]: extra space between closing brackets

**Review by mexanick — CHANGES_REQUESTED** [review]
few minor edits. otherwise looks good =)

File: src/calibration_routines.cpp (hunk 1 above)
Comment (jsturdy) [reply src/calibration_routines.cpp]: Indeed... I'll fix

File: src/calibration_routines.cpp (hunk 2 above)
Comment (jsturdy) [reply src/calibration_routines.cpp:1103]: Error was not introduced in this PR

File: src/calibration_routines.cpp (hunk 3 above)
Comment (jsturdy) [reply src/calibration_routines.cpp:907]: Error was not introduced in this PR

File: src/calibration_routines.cpp (hunk 4 above)
Comment (jsturdy) [reply src/calibration_routines.cpp:899]: Error was not introduced in this PR

File: src/daq_monitor.cpp (hunk 5 above)
Comment (jsturdy) [reply src/daq_monitor.cpp:280]: Error was not introduced in this PR

File: src/daq_monitor.cpp (hunk 6 above)
Comment (jsturdy) [reply src/daq_monitor.cpp:303]: Error was not introduced in this PR

**Comment by jsturdy** [comment]
Fixed the error introduced, did not touch the persisting style errors.
There are plenty of other style issues that need cleanup (inconsistent spacing, overabundance of comments for control flow, `if/else` block style consistency, etc.) that are not addressed here.

**Review by mexanick — APPROVED** [review]
Ok. I mentioned few other style errors as I see you've started to hit them =) 
I agree, they should be addressed in a separate PR and before doing that we should publish coding style guidelines and eventually attach a cpp/py lint to the CI. Actually, I see that certain changes you've introduced aren't following google code style (nothing bad, just different), so I believe we really need to publish our style conventions.

**Comment by jsturdy** [comment]
@lmoureaux started to put my comments into the `cmsgemos` `CONTRIBUTING.md` guidelines when he started work on the `gemonlinedb` package; and as mentioned in a few threads, my style choices are a semi-random mish-mash of various styles :-D
We can try to standardize those more.

**Pull request merged**